tests/Benchmarks/HttpParserBench.cs

```diff
@@ -271,11 +271,11 @@ static int FullRequestLegacy()
 
 class Request : IHttpHeadersHandler, IHttpRequestLineHandler
 {
-    public Http.Method Method;
-    public Http.Version Version;
-    public string Path;
-    public string Query;
-    public string Target;
+    //public Http.Method Method;
```

Comment:
Why comment these out and leave them rather than completely removing them? This goes for the code that is commented out within this class (and the one underneath it). It adds no real value to leave it there.